src/types.rs

```diff
@@ -156,12 +158,11 @@ impl Point {
     /// Returns the distance between two points:
     ///
     /// ```
-    /// use geo::Point;
+    /// use geo::{COORD_PRECISION, Point};
     ///
     /// let p = Point::new(-72.1235, 42.3521);
     /// let dist = p.distance_to(&Point::new(-72.1260, 42.45));
-    ///
-    /// assert!(dist < 1e-1)
+    /// assert!(dist < COORD_PRECISION)
     /// ```
     pub fn distance_to(&self, point: &Point) -> f64 {
         ((self.x() - point.x()).powi(2) + (self.y() - point.y()).powi(2)).sqrt()
```

Comment:
I didn't see it before, but it's work well with a Cartesian coordinate system, but it's not our case. in kraken we use this: https://en.wikipedia.org/wiki/Haversine_formula